cookbooks/aws-parallelcluster-scheduler-plugin/files/default/compute_fleet_status/update-compute-fleet-status.py

```diff
@@ -0,0 +1,98 @@
+import argparse
```

Comment:
think about having just one python script, that has a parameter --action get/update.
This would simplify testing and avoid code duplication

Reply:
Good idea! Change to one script